app/frontend/src/search/ui/sort.js

```diff
@@ -0,0 +1,32 @@
+export const renderSortSelect = (renderOptions, isFirstRender) => {
+    const {
```

Comment:
Alphabetise, please.

Reply:
this is a pain to address without making this quite unreadable as these arguments are supplied from instantsearch.js to any widget